test/verify/check-networking-firewall

```diff
@@ -56,21 +56,21 @@ class TestFirewall(NetworkCase):
         self.wait_onoff("#networking-firewall-switch", True)
         wait_unit_state(m, "firewalld", "active")
 
-        active_rules = get_active_rules(m)
-        b.wait_in_text("#networking-firewall-summary", "{} Active Rule".format(len(set(active_rules))))
+        active_zones = len(m.execute("firewall-cmd --get-active-zones | grep '^[a-zA-Z]'").split())
+        b.wait_in_text("#networking-firewall-summary", "{} Active Zone".format(active_zones))
 
         self.toggle_onoff("#networking-firewall-switch")
         self.wait_onoff("#networking-firewall-switch", False)
         wait_unit_state(m, "firewalld", "inactive")
-        b.wait_in_text("#networking-firewall-summary", "0 Active Rules")
+        b.wait_in_text("#networking-firewall-summary", "0 Active Zone")
```

Comment:
Your previous version with "0 Active Zones" was right -- in English (and many other languages), plural is being used for "0 things".